### DISCUSSION TITLE
Lugar de donde irán los pagos.

### DISCUSSION THREAD

**Discussion by SaryNotfound** [Curcolor/COMMUNITY#13] [Q&A] [answered]
Quisiera saber la opinión de ustedes en cuanto al lugar de donde se colocará la sección de pagos, tengo 2 opciones que son.

1. Colocarla en la configuración el destino de los recaudos (cabe resaltar que en ningún momento comMunity recibirá algo de los pagos que se realizan, si no la cuenta a la que irán los fondos, de las personas que necesitan ayuda claramente).
2. En la descripción del anuncio de la persona que necesite la ayuda se coloca por ejemplo el link de PayPal para que le hagan le lleguen los fondos ahí.

Pienso que la segunda opción estaría mejor, ya que en términos de programación y en los conocimientos que tenemos, se nos hace bastante complicado y más con el corto tiempo que tenemos.

De igual manera les dejo abierta esta pregunta para saber que opinan.

**Comment by JUNIORRDSR** [comment, marked as answer]
> [!TIP]
> opino que solo sea en la descripción según lo que dices, pero en caso de realizar peticiones de donaciones a causas será mejor recomendar al usuario entrar a las paginas que presentan una mejor organización en términos de recaudación de fondos
